clap_derive/tests/default_value.rs

```diff
@@ -5,10 +5,10 @@ mod utils;
 use utils::*;
 
 #[test]
-fn auto_default_value() {
+fn default_value() {
     #[derive(Clap, PartialEq, Debug)]
     struct Opt {
-        #[clap(default_value)]
+        #[clap(default_value = "0")]
```

Comment:
Can you use something other than `0` or `1` here?

Reply:
What is it you feel missing with using 0/1 that I should cover?

For the raw case, "0" is non-default.  1 would never show up from defaults.

Reply:
It's just we are testing for `0` & `1` and I wanted this to be `"2"`